modules/pam_setquota/pam_setquota.8.xml

```diff
@@ -0,0 +1,239 @@
+﻿<?xml version="1.0" encoding='UTF-8'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+	"http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd">
+
+<refentry id="pam_setquota">
+
+  <refmeta>
+    <refentrytitle>pam_setquota</refentrytitle>
+    <manvolnum>8</manvolnum>
+    <refmiscinfo class="sectdesc">Linux-PAM Manual</refmiscinfo>
+  </refmeta>
+
+  <refnamediv id="pam_setquota-name">
+    <refname>pam_setquota</refname>
+    <refpurpose>PAM module to set or modify disk quotas on session start</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <cmdsynopsis id="pam_setquota-cmdsynopsis">
+      <command>pam_setquota.so</command>
+       <arg choice="opt">
+        fs=<replaceable>/home</replaceable>
+      </arg>
+       <arg choice="opt">
+        overwrite
+        </arg>
+        <arg choice="opt">
+        startuid=<replaceable>1000</replaceable>
+      </arg>
+       <arg choice="opt">
+        enduid=<replaceable>0</replaceable>
+      </arg>
+       <arg choice="opt">
+        bsoftlimit=<replaceable>19000</replaceable>
+      </arg>
+       <arg choice="opt">
+        bhardlimit=<replaceable>20000</replaceable>
+      </arg>
+       <arg choice="opt">
+        isoftlimit=<replaceable>3000</replaceable>
+      </arg>
+       <arg choice="opt">
+        ihardlimit=<replaceable>4000</replaceable>
+      </arg>
+    </cmdsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1 id="pam_setquota-description">
+
+    <title>DESCRIPTION</title>
+
+    <para>
+      pam_setquota is a PAM module to set or modify a disk quota at session start
+    </para>
+    <para>
+      This makes quotas usable with central user databases, such as MySQL or LDAP.
+    </para>
+
+  </refsect1>
+
+  <refsect1 id="pam_setquota-options">
+
+    <title>OPTIONS</title>
+    <para>
+      <variablelist>
+
+        <varlistentry>
+          <term>
+            <option>overwrite</option>
+          </term>
+          <listitem>
+            <para>
+	      Override an existing quote. Note: Enabling this will remove the ability for the admin to manually configure different quotas for users for a filesystem with <citerefentry><refentrytitle>edquota</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
+            </para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term>
+            <option>fse<replaceable>/home</replaceable></option>
+          </term>
+          <listitem>
+            <para>
+              The device file or mountpoint the policy applies to. Defaults to the filesystem containing the users home directory.
+            </para>
+          </listitem>
+	</varlistentry>
+      <varlistentry>
+          <term>
+            <option>startuid=<replaceable>1000</replaceable></option>
+          </term>
+          <listitem>
+            <para>
+              Describe the start of the UID range the policy is applied to. (Defaults to 1000)
```

Comment:
Could you please take the default from login.defs UID_MIN? We have helper function for that.

Reply:
You mean something like
`param->startuid = pam_modutil_search_key (pamh, LOGIN_DEFS, "UID_MIN");`

Reply:
https://github.com/linux-pam/linux-pam/pull/94/commits/292deb0ebcb19edf80369f0e24439e81ffbb890f should address this, unless I misunderstood you.

Reply:
Yes, I meant that.